### ISSUE TITLE
issue with Weblinks 3.6.0-beta

### ISSUE THREAD

**Issue by jcw30dev** [joomla-extensions/weblinks#299]
#### Steps to reproduce the issue
installation of Weblinks 3.6.0-beta was quick and easy
created several web links
enabled module but; did not appear
changed Menu Assignment in module to display "on all pages" (because it defaulted to "no pages")

#### Expected result
module of "Web Links" appear on home page (and other pages)

#### Actual result
my home page is now blank; all content gone; a simple view source on the page shows no code at all

#### System information (as much as possible)
fresh install of Joomla 3.6.5
no other add on's installed; only Weblinks 3.6.0-beta

#### Additional comments
if I set the Menu Assignment in the module back to "no pages" my home page works fine again
I also tried creating a web link category and I set all links I created to that category and still does not work
bottom line: I can't get the Web Links module to display on my pages and when I try it 'blanks' the page out completely

**Comment by alikon** [comment]
can you test with the latest staging ? 
the issue has been solved #284

**Comment by jcw30dev** [comment]
Yes.  Where can I find the latest staging?  I downloaded and installed Weblinks-3.6.0-beta on 1/28/2017.  Will the latest staging be newer?  Thank you.

**Comment by yvesh** [comment]
@jcw30dev we will make a new test release in the next days. :-)

**Comment by zero-24** [comment]
@jcw30dev you try the fix by applying this change: https://patch-diff.githubusercontent.com/raw/joomla-extensions/weblinks/pull/284.diff

Can you try that?

**Comment by yvesh** [comment]
@jcw30dev there is a new release candidate, can you test now?

**Comment by jcw30dev** [comment]
The new version (Web Links 3.6.0-rc1) works now (solves the issue above).  Thank you.  However, when a link is clicked it will not open in a new window even though that is what I selected in the module setting.  It opens in the same window only.

**Comment by yvesh** [comment]
@jcw30dev i can confirm this issue, the module settings are ignored. Weblinks always uses the settings from the weblink itself, which also makes more sense.

Going to remove the setting from the module.

**Comment by yvesh** [comment]
Closing as we have a PR.

@jcw30dev thanks for testing, if you find  anything else don't hesitate opening a new issue

**Comment by jcw30dev** [comment]
thank you.  works great now using the setting in each web link.
